doc/api/buffer.md

```diff
@@ -2508,6 +2508,9 @@ encoding to another. Returns a new `Buffer` instance.
 Throws if the `fromEnc` or `toEnc` specify invalid character encodings or if
 conversion from `fromEnc` to `toEnc` is not permitted.
 
+Encodings supported by `buffer.transcode()` include: `'ascii'`, `'utf8'`,
+`'utf16le'`, `'ucs2'`, `'latin1'`, and `'binary'`.
```

Comment:
Does `'hex'` needs to be added here? ([documentation](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings))

Reply:
No. 'hex' is not supported